src/Strings.ts

```diff
@@ -315,6 +330,8 @@ export const strings = (locale: string) => {
         locale = 'zh-hans';
     else if (locale.startsWith('cs'))
         locale = 'cs-cz';
+    else if (locale === 'ja')
+        locale = 'ja';
```

Comment:
This code literally does nothing, so there's no reason to add it.

Many other locales have regional variations. For instance, American English has minor spelling variations from British English. This pattern allows us start out returning a single set of strings for a locale, then add regional variations later if necessary.

If Japanese has no regional variations, then you can just provide a single set of strings and not add this pattern.

Reply:
@billba 
Thank you for reviewing my changes. I understood that I don't need to add `ja`.
But, I found a problem. If locale is not explicitly checked, it will always be `en-us`.
So, I'll fix the problem.